fido/fido.py

```diff
@@ -5,7 +5,6 @@
 import os
 from urlparse import urlparse
 
-import concurrent.futures
 import crochet
 from twisted.internet import reactor
```

Comment:
importing reactor is something we know can lead to problems because it leads to https://github.com/itamarst/crochet/issues/45 like issues